setup.py

```diff
@@ -0,0 +1,33 @@
+from setuptools import setup
+
+package_name = 'mock_ros2_project'
+
+setup(
+    name=package_name,
+    version='0.0.1',
+    packages=[package_name],
+    data_files=[
+        ('share/ament_index/resource_index/packages',
+            ['resource/' + package_name]),
+        ('share/' + package_name, ['package.xml']),
+    ],
+    install_requires=['setuptools'],
```

Comment:
Only `setuptools` is declared as a runtime dependency, yet every ROS 2 node here imports at least `rclpy` (and often message packages). Omitting them means `pip install` will succeed but the executables will crash at import time. Please add the necessary ROS 2 Python libraries (e.g. `rclpy`, `std_msgs`, etc.) to `install_requires`.

```suggestion
    install_requires=['setuptools', 'rclpy', 'std_msgs', 'example_interfaces'],
```